### DISCUSSION TITLE
My Dashboard is behind by one hour

### DISCUSSION THREAD

**Discussion by chrissard2** [jlai/fitness-dashboard#81] [Q&A]
I shut down Chrome and still one hour behind.  I already had 11 am 250 steps completed but it is not showing up.
![Screenshot_20241103-120352](https://github.com/user-attachments/assets/755418b7-b0e8-41bc-a058-ac2a464ad0dc)
![dashboard](https://github.com/user-attachments/assets/8c8c470c-b810-4ab7-813e-5f0d6c53e160)

**Comment by jlai** [comment]
Wonder if the Fitbit API is getting confused by the daylight saving time transition. The dashboard just asks the server for the hourly step data for the current day (Nov 3). We don't do any time/timezone conversion. So the number of steps at 11AM is what the server says for "11:00".

If you look at the hourly step count (mobile app: tap steps, drag finger over hour to see counts; dashboard: click the steps tile and view the details tab) does it look like it's offset by an hour?

**Reply by chrissard2** [reply]
yes it is off by one hour.

**Comment by chrissard2** [comment]
It is correct today.

**Reply by jlai** [reply]
Yeah sounds like the Fitbit API just doesn't handle daylight saving well.
